pages/docs/tutorials/generating-code.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Streetlights - Generating Code 
+description: In this tutorial, you'll learn how to generate your code from an AsyncAPI document.
+weight: 130
+---
+
+# Introduction
+
+In this tutorial, you'll learn how to generate your code from an AsyncAPI document using the AsyncAPI generator
+
+# Background Context
+The [AsyncAPI Generator](https://github.com/asyncapi/generator) tool is used to create the documentation from the specification file. It is built in JavaScript and can be accessible using npm or Docker. 
+
+The generator tool supports a number of templates to generate code for a variety of different languages and protocols as the output. These templates helps to specify what exactly must be generated and for this tutorial, a Nodejs template will be specified.
```

Comment:
1. Both `you` and `we` are great to use in a tutorial because they accomplish different useful goals (which Lukasz already mentioned.) That said, I agree that consistency is key and it's always nice to choose one and stick to that selection. I think using `you` is great in our tutorials, but I don't want it to be "wrong" to sometimes use the word "we". 😂 

2. Yes, tutorial steps should be actions taken in **present** tense.